src/Components/TaskForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react'
+
+import {
+    Box,
+    Button,
+    Container,
+    InputLabel,
+} from '@material-ui/core';
+
+import { Formik, Form, Field, ErrorMessage } from 'formik'
+
+import schema from './Validation/schema';
+
+interface Props {
+    onTaskAdded: Function
+}
+
+export interface ITask {
+    title: string
+}
+
+interface IMethods {
+    resetForm: Function
+}
+
+const TaskForm: FC<Props> = ({ onTaskAdded }: Props) => {
+
```

Comment:
I suggest removing this line.

Reply:
Resolved on 817a540bae01e32c9eebee2b77b9a51ce9bf8437